README.md

```diff
@@ -1,3 +1,20 @@
 ## Welcome to the .NET SDK Workload Versions repo
 
-This repository contains the version information for .NET SDK Workloads.
\ No newline at end of file
+This repository contains the version information for .NET SDK Workloads.
+
+### Pre-requisites for local VS insertion build
+
+1. Install the latest [Visual Studio](https://visualstudio.microsoft.com/downloads/) with the .NET Desktop workload
+  - Make sure to restart your PC after the installation is complete.
+2. [Install Azure CLI](https://learn.microsoft.com/cli/azure/install-azure-cli-windows#install-or-update)
+3. Run `az login` to authenticate
+  - When it asks for a subscription to select, just press Enter. The default subscription selection does not affect DARC.
+4. [Install DARC](https://github.com/dotnet/arcade/blob/main/Documentation/Darc.md#setting-up-your-darc-client)
+5. [Add GitHub auth for DARC](https://github.com/dotnet/arcade/blob/main/Documentation/Darc.md#step-3-set-additional-pats-for-azure-devops-and-github-operations)
+  - Use the [darc authenticate](https://github.com/dotnet/arcade/blob/main/Documentation/Darc.md#authenticate) command.
+  - Generate the GitHub PAT [here](https://github.com/settings/tokens?type=beta). Create a fine-grained PAT instead of the classic PAT.
+  - **Do not** create an AzDO PAT. Leave that entry blank in the darc-authenticate file for it to use local machine auth.
+6. Request access to the [.NET Daily Internal Build Access](https://coreidentity.microsoft.com/manage/Entitlement/entitlement/netdailyinte-q2ql) entitlement
+  - This allows the local AzDO machine auth to gather internal assets from AzDO.
+  - ***Message Matt Mitchell*** (@mmitche) for approval after requesting access to the entitlement.
```

Comment:
should we publicly list alias and names? Seems like these docs should be internal only. They do not apply to any external user

Reply:
I would normally just use the public GH handle but I do know he has his full name in his profile currently. That could change in the future.

Reply:
I've removed his name and directed them to check the list of primary owners.